src/Aspire.ProjectTemplates/README.md

```diff
@@ -0,0 +1,81 @@
+      
+      **Note:** There's a few ways to determine the static version value:
+      - Look at the contents of the latest released version of the templates package at https://nuget.info/packages/Aspire.ProjectTemplates and find the version from the relvant *.csproj* file in the template package content
+      - Checkout the relevant `release/X.X` branch for the latest public release, e.g. `release/9.4`, and in the *./src/Aspire.ProjectTemplates/* directory, run the `dotnet` CLI command to extract the appropriate version from the build system, e.g. `dotnet msbuild -getProperty:MicrosoftAspNetCorePackageVersionForNet9`. The property name to pass for a given replacement token can be determined by looking in the *./src/Aspire.ProjectTemplates/Aspire.ProjectTemplates.csproj* file, at the `<WriteLinesToFile ...>` task, which should look something like the following:
+         ```xml
+         <WriteLinesToFile File="%(TemplateProjectFiles.DestinationFile)"
+                           Lines="$([System.IO.File]::ReadAllText('%(TemplateProjectFiles.FullPath)')
+                                    .Replace('!!REPLACE_WITH_LATEST_VERSION!!', '$(PackageVersion)')
+                                    .Replace('!!REPLACE_WITH_ASPNETCORE_9_VERSION!!', '$(MicrosoftAspNetCorePackageVersionForNet9)')
+                                    .Replace('!!REPLACE_WITH_ASPNETCORE_10_VERSION!!', '$(MicrosoftAspNetCorePackageVersionForNet10)')
+                                    .Replace('!!REPLACE_WITH_DOTNET_EXTENSIONS_VERSION!!', '$(MicrosoftExtensionsHttpResilienceVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_LTS_VERSION!!', '$(OpenTelemetryLTSVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_EXPORTER_VERSION!!', '$(OpenTelemetryExporterOpenTelemetryProtocolVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_HOSTING_VERSION!!', '$(OpenTelemetryInstrumentationExtensionsHostingVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_ASPNETCORE_VERSION!!', '$(OpenTelemetryInstrumentationAspNetCoreVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_HTTP_VERSION!!', '$(OpenTelemetryInstrumentationHttpVersion)')
+                                    .Replace('!!REPLACE_WITH_OTEL_RUNTIME_VERSION!!', '$(OpenTelemetryInstrumentationRuntimeVersion)') )"
+                           Overwrite="true" />
+         ```
+
+8. Updating the versions for non-Aspire packages referenced in all *.csproj* files in the content folder named for the new latest version, e.g. *./9.5/**/*.csproj*, isn't covered as part of this process. These package versions should be updated by our regular process for updating the versions of our dependencies.
+
+For the `aspire-empty` template:
+
+1. Edit *./aspire-empty/.template.config/template.json* and ensure the tags contains correct values to encode valid TFMs for template's supported Aspire version, e.g.:
```

Comment:
AI presamably wouldn't be able to guess if eg it's time to add net11.0. we'd have to tell it.

Reply:
Yep, we make that judgement manually